packages/lms/src/lib/styles/_components.scss

```diff
@@ -14,11 +14,11 @@
   cursor: pointer;
 }
 
-.icon-f {
+.material-icons {
   @include icon;
 }
 
-.icon-o {
+.material-icons {
   @include icon;
   font-variation-settings: 'FILL' 0;
 }
```

Comment:
This isn't really DRY compliant. Also, shouldn't these perhaps be variance options as props in the component? Like you can then use the component like so `<Icon name="home" variant="filled" />` and it'd give you a filled icon?

Reply:
That's a nice idea but might be a good idea for a new task